src/styles/card.scss

```diff
@@ -0,0 +1,153 @@
+* {
+  box-sizing: border-box;
+  background-color: #fff;
+}
+
+p,
+h3 {
+  margin: 0;
+}
+
+body {
+  margin: 0;
+  font-family: "Roboto", sans-serif;
+}
+
+.card-container {
+  display: grid;
+  grid-template-columns: repeat(4, 1fr);
+  grid-gap: 48px 46px;
+  max-width: 1200px;
+  margin: 0 auto;
+  padding: 48px;
+  top: 50px;
+  bottom: 72px;
+}
+
+@media (max-width: 1023px) {
+  .card-container {
+    grid-template-columns: repeat(3, 1fr);
+  }
+}
+
+@media (max-width: 767px) {
+  .card-container {
+    grid-template-columns: repeat(2, 1fr);
+  }
+}
+
+@media (max-width: 487px) {
+  .card-container {
+    grid-template-columns: 1fr;
+  }
+}
+
+.card {
+  width: 200px;
+  padding: 32px 16px 16px;
+  display: flex;
+  flex-direction: column;
+  border-radius: 5px;
+  border: 1px solid #f3f3f3;
+  background: #fff;
+
+  &__img {
+    width: 160px;
+    height: 134px;
+    margin: 0 auto 40px;
+  }
+
+  &__name {
+    margin-bottom: 4px;
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    color: #060b35;
+  }
+
+  &_code {
+    margin-bottom: 16px;
+    line-height: 14px;
+    font-size: 10px;
+    color: #616070;
+  }
+
+  &__raiting {
+    margin-bottom: 24px;
+    display: flex;
+    justify-content: space-between;
+    align-items: center;
+  }
+
+  &__raiting-review {
+    font-size: 10px;
+    line-height: 14px;
+  }
+
+  &__price {
+    margin-bottom: 16px;
+    display: flex;
+    font-size: 12px;
+    line-height: 16px;
+    justify-content: space-between;
+    color: #616070;
+  }
+
+  &__price-cost {
+    font-weight: 700;
+    line-height: 18px;
+    font-size: 16px;
+    color: #060b35;
+  }
+
+  &__price-text {
+    margin: 0;
+    font-size: 12px;
+    line-height: 18px;
+  }
+
+  &__button {
+    height: 40px;
```

Comment:
don not hardcode height,  use proper paddings instead

Reply:
not fixed yet